### ISSUE TITLE
Expecting checkpoint >= 1. Waiting. | After one lap in the simulator

### ISSUE THREAD

**Issue by JustinGuese** [aws-deepracer-community/deepracer-core#69]
Error log of crr0004/deepracer_robomaker:console:
`SIM_TRACE_LOG:19,102,2.9274,0.7746,-0.8749,0.52,0.40,8,0.0000,True,False,1.9954,292,69.25,1570613566.0205994

reward: 50.63380000999996
Training> Name=main_level/agent, Worker=0, Episode=20, Total reward=49.63, Steps=3038, Training iteration=0
Expecting checkpoint >= 1. Waiting.
Expecting checkpoint >= 1. Waiting.
Expecting checkpoint >= 1. Waiting.
Expecting checkpoint >= 1. Waiting.
Expecting checkpoint >= 1. Waiting.
Expecting checkpoint >= 1. Waiting.
`

Anyone experienced this issue?
After one lap the car patiently waits at the former start position, resulting in the error log above.
I have read that it might relate to the tensorflow setup? Is it possible to change it somehow?

**Comment by daj** [comment]
I would guess this is intentional because the training is waiting for a checkpoint to be made before it can continue.

The [code comment](https://github.com/crr0004/deepracer/blob/ae3e571d821553a74e94db4438577ac7f03989cd/rl_coach/src/markov/s3_boto_data_store.py#L191) for the `Expecting checkpoint >= X. Waiting.` output says:
```  
 # if we get a checkpoint that is older that the expected checkpoint, we wait for
 #  the new checkpoint to arrive.
```

FYI, the [Amazon DeepRacer architecture docs](https://docs.aws.amazon.com/deepracer/latest/developerguide/deepracer-how-it-works-service-architecture.html) show this architecture diagram, which shows you that training involves running simulations in RoboMaker's virtual environment and feeding the results to SageMaker:

![deepracer-how-it-works-architecture](https://user-images.githubusercontent.com/739125/68999523-2b6b0180-088f-11ea-9d30-fa084b839aaf.png)

The [Amazon DeepRacer solution workflow docs](https://docs.aws.amazon.com/deepracer/latest/developerguide/deepracer-how-it-works-solution-workflow.html) mention

>The AWS DeepRacer service periodically saves the neural network model to persistent storage.

So it's not surprising to me that you see these pauses, especially if you are doing local training on a lower specification machine.

**Comment by breadcentric** [comment]
If it keeps waiting indefinitely, most likely sagemaker has failed to perform the training and create the checkpoint
